pandas/core/tools/datetimes.py

```diff
@@ -379,11 +381,12 @@ def _convert_listlike(arg, box, format, name=None, tz=tz):
             raise TypeError('arg must be a string, datetime, list, tuple, '
                             '1-d array, or Series')
 
-        arg = _ensure_object(arg)
+        obj_arg = _ensure_object(arg)
```

Comment:
Since I was trying to have `_convert_listlike` handle `Series`, I was using the `arg` variable to keep track if a `Series` was initially passed so I know to use `dt.tz_localize()` after the dates have been parsed. `arg = _ensure_object(arg)` converts `arg` from a `Series` to a `np.array` so I no longer know what data structure was originally passed. 

I guess I could create a boolean variable `arg_is_series`in the beginning and keep track of this information that way.